WordPress/Resources/ar.lproj/Localizable.strings

```diff
@@ -185,7 +185,7 @@
 "%ld files successfully uploaded" = "تم رفع %ld من الملفات بنجاح";
 
 /* Displays the number of words and characters in text */
-"%li words, %li characters" = "%1$lكلمات i و%2$lحروف i";
+"%li words, %li characters" = "%1$li كلمات، %2$li أحرف";
```

Comment:
ℹ️ Note: this is one of those example cases where bidi-text in Unicode made it quite confusing to fix (and for which I'm so happy the linter is there to figure out for us if this is correct or not).

For example, in this string here, on GitHub's web UI it will visually appear as `"%1$li 2%<arabic text>$li<arabic text>"` which feels odd and wrong, with the 2nd placeholder looking incorrect.

But it's actually not and is correct: it's just looking like this on GitHub UI because of bidi algorithm, but if you [use a utf8-dissect tool](https://mc.a8c.com/i18n/tools/utf8-dissect.php?query=%22%251%24li+%D9%83%D9%84%D9%85%D8%A7%D8%AA%D8%8C+%252%24li+%D8%A3%D8%AD%D8%B1%D9%81%22) you can see that the raw characters are in the right order and that this odd rendering before placeholder resolution is because of the [mix of characters with different unicode bidi properties (`$`/`%` punctuation vs `2` number characters vs `li` latin text characters vs arabic text, all having different characteristics](https://href.li/?https://www.w3.org/International/articles/inline-bidi-markup/uba-basics) and making the rendering look odd, especially until the placeholder is replaced by an actual value)

Reply:
> this odd rendering before placeholder resolution is because of the mix of characters with different unicode bidi properties ($/% punctuation vs 2 number characters vs li latin text characters vs arabic text, all having different characteristics

My head is spinning. Great detective work!